app/src/main/java/de/westnordost/streetcomplete/quests/shop_type/CheckShopExistence.kt

```diff
@@ -0,0 +1,60 @@
+package de.westnordost.streetcomplete.quests.shop_type
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.elementfilter.toElementFilterExpression
+import de.westnordost.streetcomplete.data.osm.geometry.ElementGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmElementQuestType
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.user.achievements.EditTypeAchievement.CITIZEN
+import de.westnordost.streetcomplete.osm.IS_SHOP_EXPRESSION
+import de.westnordost.streetcomplete.osm.IS_SHOP_OR_DISUSED_SHOP_EXPRESSION
+import de.westnordost.streetcomplete.osm.LAST_CHECK_DATE_KEYS
+import de.westnordost.streetcomplete.osm.Tags
+import de.westnordost.streetcomplete.osm.isShopExpressionFragment
+import de.westnordost.streetcomplete.osm.updateCheckDate
+import de.westnordost.streetcomplete.quests.existence.CheckShopExistenceForm
+
+class CheckShopExistence : OsmFilterQuestType<Unit>() {
+
+    // opening hours quest acts as a de facto checker of shop existence, but some people disabled it.
+    // separate from CheckExistence as very old shop with opening hours should show
+    // opening hours resurvey quest rather than this one (which would cause edit date to be changed
+    // and silence all resurvey quests)
+
+    override val elementFilter = """
+        nodes with (
+          (${isShopExpressionFragment()})
+             and !man_made
+             and !historic
+             and !military
+             and !power
+             and !attraction
+             and !aeroway
+             and !railway
+             and amenity !~ clinic|library|music_school|post_office
+             and healthcare !~ clinic
```

Comment:
Why not?

Reply:
rechecking these every two years seems a bit too much, but that may be assuming too stronly that my area is too typical and this kind of places can be far more unstable elsewhere

Reply:
But it is just a simple yes/no question. IMO it's okay and not worth the extra brainpower to try to maintain a balance here. (After all, if you keep the exclusion here, the next thing I or someone else would request is that the existence of these would also be asked but less frequently.)

Reply:
I was rather thinking about risk of someone making mistake and answering incorrectly.

I thought about asking them with longer timeout and I can add this.

Reply:
How would asking with longer timeouts solve the issue you note? Such larger facilities seem more difficult to miss, too. So I do not really see this issue.

Reply:
Some `amenity=clinic` are not very prominently signed (I encountered them during testing and were initial ones added, that is how I ended asking you about this hospitals and doing https://github.com/streetcomplete/StreetComplete/commit/fd0a1c3c0853fc1caf3e96e35d71f971f9ba3c69 )

Reply:
Well, but still, asking less often or even not asking at all in this quest is not going to solve this. After all, the opening hours of clinics are also asked - every year. And most users will have the opening hours quest activated.

Reply:
OK, I removed special handling of major objects.